rotkehlchen/accounting/history_base_entries.py

```diff
@@ -125,7 +150,7 @@ def process(
             )
 
         self.pot.add_asset_change_event(
-            method=event_settings.method,
+            method=method,
```

Comment:
don't do it like that. Don't re-create the old code. Use the `EventDirection` instead here